apps/server/src/shared/domain/entity/dashboard.ts

```diff
@@ -159,6 +159,7 @@ export type GridElementWithPosition = {
 
 export type DashboardProps = { colums?: number; grid: GridElementWithPosition[]; userId: EntityId };
 
+// TODO: is not marked as Entity and should not named as Entity
```

Comment:
Ist das noch relevant?

Reply:
TODO: Kann raus, aber "Is not decorate as Entity and should not named as Entity!" Sollten wir lassen. Damit es nieman zurück baut.